hw/net/rocker/rocker_hw.h

```diff
@@ -165,6 +167,14 @@ enum {
     ROCKER_TLV_CMD_TYPE_OF_DPA_GROUP_DEL,
     ROCKER_TLV_CMD_TYPE_OF_DPA_GROUP_GET_STATS,
 
+    ROCKER_TLV_CMD_TYPE_CLEAR_PORT_STATS,
```

Comment:
Yes, I copied them from the driver code when I updated net-next. Ran into
tlv types not matching between driver and device. I think once all repos
are up to date this will not show up as a diff.

On Sun, Apr 19, 2015 at 9:08 PM, Scott Feldman <notifications@github.com>
wrote:

> In hw/net/rocker/rocker_hw.h
> <https://github.com/pbhide/qemu-rocker/pull/1#discussion_r28663127>:
>
> > @@ -165,6 +167,14 @@ enum {
> >      ROCKER_TLV_CMD_TYPE_OF_DPA_GROUP_DEL,
> >      ROCKER_TLV_CMD_TYPE_OF_DPA_GROUP_GET_STATS,
> >
> > +    ROCKER_TLV_CMD_TYPE_CLEAR_PORT_STATS,
>
> Are these STATS updates from David Ahern? I know he's been patiently
> waiting to get some rocker device stats patches in. I should just pull them
> in and update scottfeldman/qemu-rocker.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pbhide/qemu-rocker/pull/1/files#r28663127>.
>